Jenkinsfile

```diff
@@ -53,8 +53,22 @@ pipeline {
             sh "helm3 upgrade --install ${K8S_RELEASE_NAME} ./deploy/charts --namespace ${K8S_NAMESPACE} --values ./deploy/dev/values.yaml --set image.tag=${TAG_NAME} --set deployenv.environment=k8sdev"
           }
         }
-        }
+	  }
+    }
+
+    stage('Kubernetes_Test_Deployment') {
+      when {
+        branch 'master'
       }
+      steps {
+        echo "Deploying to development Kubernetes cluster..."
+        withKubeConfig([credentialsId: "jenkins-ci-kubeconfig"]) {
+          withAWS([credentials: "jenkins-ci"]) {
+            sh "helm3 upgrade --install ${K8S_RELEASE_NAME} ./deploy/charts --namespace ${K8S_NAMESPACE} --values ./deploy/test/values.yaml --set image.tag=${TAG_NAME} --set deployenv.environment=k8stest"
```

Comment:
Are you planning on deploying this to the test namespace?

Reply:
Not sure what you mean by that.

The configs referenced here are to deploy a new instance of the container to ui-test.dk1.kiva.org with specific UI configuration which will eventually be served in the context of https://www.test.kiva.org.

Reply:
I think it's that `--namespace ${K8S_NAMESPACE}` will end up deploying this to the (dev) namespace -- kiva-marketplace. So in the context of this pipeline we'd end up deploying over our dev deployment.

Reply:
Oh I see now... can update